data/_ui/interfaces.txt

```diff
@@ -425,6 +425,11 @@ interface "load menu"
 	button d "_Delete"
 		center -285 155
 		dimensions 90 30
+
+	active
```

Comment:
If you don't want to deal with modifying the side panel png files, you can actually just add a button sprite here instead. 

It would look something like this: 
```
	sprite "ui/wide button"
		center -421 195
```